app/auth/login.js

```diff
@@ -24,10 +27,35 @@ const Login = () => {
 
     return (
         <AuthPage>
-            <span>Login Screen</span>
+            <Text>Login Screen</Text>
+            <TextInput
+                label="Username"
+                value={email}
+                onChangeText={(text) => setEmail(text)}
+                style={{ width: 350 }}
+            />
+            <TextInput
+                label="Password"
+                value={password}
+                onChangeText={(text) => setPassword(text)}
+                secureTextEntry={true}
+                style={{ width: 350 }}
+            />
             <Button mode="contained" onPress={onLoginPress}>
                 Login
             </Button>
+            <Text
+                onPress={() => router.replace('/auth/forgot')}
+                style={{ color: 'blue', textDecorationLine: 'underline' }}
+            >
+                Forgot Password?
+            </Text>
+            <Text
+                onPress={() => router.replace('/auth/create')}
+                style={{ color: 'blue', textDecorationLine: 'underline' }}
```

Comment:
This duplicate style could be moved into a StyleSheet. Check the `Container` component for an example of that.

Reply:
Per https://reactnative.dev/docs/stylesheet let's keep the StyleSheet's outside of the main render function. Again, see the Container component.